### ISSUE TITLE
Headers for Pages

### ISSUE THREAD

**Issue by igbanam** [standardebooks/khalil-gibran_the-prophet#1]
Amazing work y'all did here on bringing this to Github 🥳🙇🏾‍♂️

Was reading through this and it would be nice to see the headers in the sections linked to from the Table of Contents.

Is this something the team would appreciate? I don't mind doing the work here. This has been one of my favorite books of all time.

Thank you

**Comment by acabal** [comment]
I'm not sure what you're referring to; can you be a little more specific?

**Comment by igbanam** [comment]
When you click on a chapter from the table of contents, it scrolls to the chapter in the book. But there is no header corroborating the chapter on that page. What I am proposing is to add headers for the chapters so readers know where they are. I have attached a screen recording to show what this looks like (p.s GitHub doesn't allow video files here, so I zipped it up)
[19-08-14-15-19-27.mp4.zip](https://github.com/standardebooks/khalil-gibran_the-prophet/files/3501789/19-08-14-15-19-27.mp4.zip)
